examples/volumes/main.tf

```diff
@@ -25,7 +25,7 @@ resource "oneview_volume" "volume" {
   properties {
     name              = "testvol"
     storage_pool      = data.oneview_storage_pool.st_pool.uri
-    size              = 268435456
+    size              =  element(tolist(data.oneview_storage_volume_template.st_vt.tp_size[*].default),0)
```

Comment:
I don't think this is the fix user has asked for. 
In this example you are asking user to mandatorily provide size, storage_pool, provisioning_type attributes. But user wants those attributes to be optional. It means even though he didn't provide those attributes in example, volume should be created when user provided only name and templateUri. 
I think that implementation should be done in library file to achieve the expected behavior.

Reply:
Changing the behaviour of volume  creation without giving size is asked by one user. Other user may want to provide the size.  In the example get the size from the template and user is not asked to provide the size

Reply:
ok